ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelContainer/v1/panelContainer/responsiveGrid.html

```diff
@@ -0,0 +1,27 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2022 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<template data-sly-template.responsiveGrid="${ @ container}">
+    <div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+         id="${container.id}"
+         class="cmp-container"
```

Comment:
should we have our own class also :  `class="cmp-container cmp-forms-container"`

Reply:
cmp-forms-container is for form container not panel. we can add a new cmp class, if we are ok two have two component classes on a single component.